ifStatement.js

```diff
@@ -47,6 +56,11 @@ Steps:
     - If the number is zero, print a message saying that it is zero.
 ************************************************************************************************/
 // TODO: ADD YOUR CODE BELOW
+let number=2
```

Comment:
You're initializing the number variable with a value of 2, but you should be asking the user to input the number. Therefore, you don't need to initialize number at the beginning.